dagql/objects.go

```diff
@@ -1060,6 +1067,23 @@ func (field Field[T]) ArgDeprecated(name string, paras ...string) Field[T] {
 	panic(fmt.Sprintf("field %s has no such argument: %q", field.Spec.Name, name))
 }
 
+func (field Field[T]) ArgExperimental(name string, paras ...string) Field[T] {
```

Comment:
nice - left a comment noting we'll want this too but ya beat me to it :)

Reply:
Ah, I think that while the annotation is correctly propagated, it's not actually propagating through to the codegen... yet.